.travis.yml

```diff
@@ -47,5 +48,8 @@ matrix:
   fast_finish: true
 
   allow_failures:
-    - rvm: jruby-9.1.15.0
+    - rvm: jruby-9.1.16.0
       gemfile: gemfiles/rails_52.gemfile
+    - rvm: 2.5.0
+      # devise 3.5.10 syntax incompatible error
+      gemfile: gemfiles/rails_42.gemfile
```

Comment:
Please remove these 3 lines, Rails 4.2 + ruby 2.5.0 should be passing now. Thanks!

Reply:
It seems that a wrong squash happened? This allow_failures entry is back there.

Reply:
Pfiou indeed ! Now updated, thanks.